MainActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.revontulet
+
+// Import the views package
+import com.example.revontulet.ui.screens.Revontulet
+
+import android.os.Build
+import android.os.Bundle
+import android.view.WindowManager
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.annotation.RequiresApi
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+
+@ExperimentalPermissionsApi
+@RequiresApi(Build.VERSION_CODES.S)
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        window.addFlags(WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)
```

Comment:
Mensen verschillen van mening, maar ik importeer graag zoveel mogelijk. We gebruiken ook geen lange class names.
```suggestion
        window.addFlags(FLAG_KEEP_SCREEN_ON)
```